tests/test_pds4_create_xml_index_whitebox.py

```diff
@@ -64,7 +67,7 @@ def test_load_config_object():
 
     # Tests that the config_object is loaded over.
     config_object = tools.load_config_file(
-        specified_config_files=[str(expected_dir/'tester_config.yaml'),])
+        specified_config_files=[str(expected_dir/'tester_config_nillable.yaml'),])
```

Comment:
Not required, but we usually put spaces around operators like this and you did elsewhere in the code.